plugin/storage/integration/token_propagation_test.go

```diff
@@ -40,16 +36,6 @@ type esTokenPropagationTestHandler struct {
 }
 
 func (h *esTokenPropagationTestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	// Return the elasticsearch version
```

Comment:
Was this removed because it was a temporary workaround when the version wasn't available?

Reply:
yes